docs/development.md

```diff
@@ -161,3 +161,22 @@ After installing the pre-commit requirements you can initialize pre-commit.
 pre-commit install
 pre-commit install-hooks
 ```
+
+### Changelog
+
+We're adding [Artifact Hub annotations](https://artifacthub.io/docs/topics/annotations/helm/) to our charts to improve the visibility of changes for our charts. These annotations are added to `Chart.yaml` as a YAML string and need to be replaced on every update, as they are not additive but specific to this release.
+
+```yaml
+annotations:
+  artifacthub.io/changes: |
+    - kind: changed
+      description: "feat(argo-rollouts): Upgrade to argo-rollouts 1.3.0"
```

Comment:
is this a good example? it's lazy copypasta from upstreams git history but also highly redundant. maybe our example could be more concise:
```suggestion
      description: "feat(argo-rollouts): Update to 1.3.0"
```
or even just this:
```suggestion
      description: "argo-rollouts: Update to 1.3.0"
```

Reply:
What do you think about the examples now?